concepts/05 Widgets/Form/01 Getting Started with Form/50 Configure Item Labels.md

```diff
@@ -0,0 +1,187 @@
+Simple item [labels](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/label/) display the formatted field names of the [formData](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#formData) object. You can change the displayed text in the **label**.[text](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/label/#text) property. 
+
+The Form displays labels on the left side of editors. The text of all editors is aligned to the left. To change location of all labels, use the [labelLocation](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#labelLocation) property. Use **label**.[location](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/label/#location) property to relocate individual labels. You can change the horizontal alignment of individual labels in the [alignment](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/label/#alignment) property.
+
+Use the [showColonAfterLabel](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#showColonAfterLabel) property to display colons at the end of labels. Use the **label**.[showColon](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/label/#showColon) property for the same purpose for an individual label. Individual settings override common settings.
+
+The following code shows how to locate all labels on top of editors. We align the `Notes` item label's text to the center. We also hide the colons after all labels:
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#formContainer").dxForm({
+            formData: {
+                name: "John Heart",
+                position: "CEO",
+                hireDate: new Date(2012, 4, 13),
+                officeNumber: 901,
+                notes: "John has been in the Audio/Video industry since 1990."
+            },
```

Comment:
Is it necessary to display the formData object in each section? It draws the attention away from the described properties.